index.html

```diff
@@ -1,54 +1,269 @@
+  <title>Mobily First</title>
+
+  <link rel="stylesheet" href="style.css" />
+
+  <script src="https://kit.fontawesome.com/26c38390b4.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+
+  
     <header class="navbar">
       <a href="#" class="brand-title">My logo</a>
-      <i class="fas fa-bars"></i>
+      <img src="./images/Union.png" alt="">
     </header>
 
-    <section class="presentation">
-      <h2>I’m Alexandra. Glad to see you!</h2>
-    </section>
-
-    <article class="job">
-      <p>
-        I’m a software developer! I can help you build a product , feature or
-        website Look through some of my work and experience! If you like what
-        you see and have a project you need coded, don’t hesitate to contact me.
-      </p>
-    </article>
-    <section class="contact">
-      <p>LET’S CONNECT</p>
-    </section>
-    <section class="socialmedia">
-      <ul>
-        <li><i class="fa-brands fa-twitter"></i></li>
-        <li><i class="fa-brands fa-linkedin"></i></li>
-        <li><i class="fa-solid fa-m"></i></li>
-        <li><i class="fa-brands fa-square-github"></i></li>
-        <li><i class="fa-regular fa-hand-peace"></i></li>
-      </ul>
-    </section>
-  </body>
-</html>
+    <div class="seconback">
+      <section class="presentation">
+        
+        <h1><Strong>I’m Gonzalo. Glad to see you!</Strong></h1>
+
+        <article class="job">
+          <p>
+            I’m a software developer! I can help you build a product , feature or
+            website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hesitate to contact me.
+          </p>
+        </article>
+        <section class="contactme">
+          <p>LET’S CONNECT</p>
+        </section>
+        <section class="socialmedia">
+          <ul>
+            <li class="twitter"><img src="./images/twitter.png" alt=""></li>
+            <li><img src="./images/Linkedin icon.svg" alt="">            </li>
+            <li><img src="./images/m.png" alt=""></li>
+            <li><img src="./images/git.png" alt=""></li>
+            <li><img src="./images/hand.png" alt=""></li>
+          </ul>
+        </section>
+      </section>
+
+      <section class="cards">
+        
+        <div class="card">
+
+          <img class="cardimg" src="./images/hischool.png" alt="">
+
+          <div class="grid-item-title-1">
+            <p class="title">Tonic</p>
+            <div class="canopy">
+              <p>CANOPY</p>
+              <ul>
+                <li> Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+            <article>A daily selection of privately personalized reads;
+              no accounts or sign-ups required.</article>
+          </div>
+
+          <section class="button">
+            <ul>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+            </ul>
```

Comment:
- [ ] I appreciate the effort you made while implementing the changes suggested by the previous reviewer. 😃🙌👍 Just a little more effort is further required from your side to make the technologies **tags** size according to figma design, as shown below:

#### Figma design:
![f3](https://user-images.githubusercontent.com/105105262/229159537-d79aee6b-17c7-4bde-b6bf-153a6774da94.PNG)

#### Your design:
![y3](https://user-images.githubusercontent.com/105105262/229159573-c8789879-b961-42a8-8f59-002a097fb32d.PNG)

As you can see the button sizes are according to their content, so please adjust that. Please always remember replicating exact Figma templates gives confidence to recruiters in your ability to replicate business requirements as regards design 😄